choose-native-plants/release-values.yaml

```diff
@@ -1,6 +1,6 @@
 app:
   image:
-    tag: "1.0.2"
+    tag: "v1.0.3"
```

Comment:
@boutell the docker image tags don't have the `v` prefix: https://github.com/CodeForPhilly/pa-wildflower-selector/pkgs/container/pa-wildflower-selector%2Fapp